.eslintrc

```diff
@@ -625,6 +625,14 @@
       "globals": {
         "tinymce": "readonly"
       }
+    },
+    {
+      "files": [
+        ".storybook/main.cjs"
+      ],
+      "rules": {
+        "node/no-extraneous-require": "off"
+      }
```

Comment:
Lint was flagging this line. Do you think there's a better fix here? https://github.com/GoogleForCreators/web-stories-wp/pull/5792/files#diff-b185a1de343f74ed35c31ccba4fa43156a89611a58a180bcdc8de9646284f053R71

![image](https://user-images.githubusercontent.com/22185279/153492493-f7bfc287-fc72-47bf-819c-48c28d75d042.png)

Reply:
If it's just that one line I'd probably just add an `eslint-disable-next-line` comment there instead of adding this config to this config file.